src/components/Selectors/CheckBox/checkbox.module.scss

```diff
@@ -1,141 +1,182 @@
 .selector {
-    margin-bottom: $selector-margin-bottom;
-    flex-shrink: 1;
-    position: relative;
     display: flex;
-    text-overflow: elipses;
-    vertical-align: baseline;
+    position: relative;
+    width: fit-content;
 
     input {
         position: absolute;
-        background: none;
         opacity: 0;
-        height: $selector-input-width;
-        width: $selector-input-height;
-        top: $space-xxxs;
-        left: $space-xxxs;
+        height: 100%;
+        width: 100%;
         cursor: pointer;
-    }
-
-    input:checked + label {
-        .checkmark {
-            background-color: var(--primary-color);
-            border: $space-xxxs solid var(--primary-color);
+        z-index: 1;
 
-            &.disabled {
-                opacity: 50%;
+        &[disabled] {
+            cursor: default;
 
-                &:hover {
-                    background-color: var(--primary-color);
-                    border: $space-xxxs solid var(--primary-color);
+            & + label {
+                .checkmark {
+                    opacity: 50%;
                 }
+            }
 
-                &:focus,
-                &:focus-visible {
+            &:focus + label,
```

Comment:
please remove `&:focus + label,` and move` &:focus-visible + label ...` inside `:global(.focus-visible) { ... }` using the same css path as before so that the focus outline/border only appears whilst using keyboard. Please do this for any other related focus styling.